core/src/block.rs

```diff
@@ -208,7 +208,7 @@ impl<'x> OpenBlock<'x> {
 
     /// Populate self from a header.
     fn populate_from(&mut self, header: &Header) {
-        self.block.header.set_score(*header.score());
+        self.block.header.set_score(U256::default());
```

Comment:
You can write it as `self.block.header.set_score(Default::default());` when the compiler can infer the type.
It's just a comment for further work, you don't need to fix it.